Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -59,6 +80,9 @@ def __init__(self, tribler_session):
 
         self.torrents = {}
 
+        # TODO: merge with self.torrents. Legacy stuff moved from Session
```

Comment:
Can't we just do this now? Seems pretty straightforward.

Reply:
The intention of this PR was to take the basic, most literary steps to remove LaunchMany and move stuff from Session. There are lots of opportunities to optimize stuff further in smaller PRs.